api-tests/features/inherited-identity.feature

```diff
@@ -0,0 +1,36 @@
+Feature: Inherited Identity
```

Comment:
Might want to tag the Feature (not the scenarios) with `@Build` - assuming they work against build?